src/EventListener/SendNotificationMessageListener.php

```diff
@@ -18,53 +18,51 @@
 
 class SendNotificationMessageListener
 {
-
     public function __invoke($objMessage, &$arrTokens, $language, $objGatewayModel): bool
     {
         $arrTokens2 = [];
 
-        foreach($arrTokens as $key=>$value){
-            $chunks = explode('_',$key);
-            if(empty($chunks)){
+        foreach ($arrTokens as $key => $value) {
+            $chunks = explode('_', $key);
+            if (empty($chunks)) {
                 continue;
             }
 
-            if('form' === $chunks[0]){
+            if ('form' === $chunks[0]) {
                 unset($chunks[0]);
-                $formFieldKey = implode('',$chunks);
+                $formFieldKey = implode('', $chunks);
                 $arrTokens2[$formFieldKey]['value'] = $value;
-            }elseif('formlabel' === $chunks[0]
-            ){
+            } elseif ('formlabel' === $chunks[0]
+            ) {
                 unset($chunks[0]);
-                $formFieldKey = implode('',$chunks);
+                $formFieldKey = implode('', $chunks);
                 $arrTokens2[$formFieldKey]['label'] = $value;
             }
         }
 
-        foreach($arrTokens2 as $fieldName => $fieldDefinition){
-            $objFormField = FormFieldModel::findOneBy(['pid = ?','name = ?'],[$arrTokens['formconfig_id'],$fieldName]);
-            if(!$objFormField
-            || $objFormField->is_technical_field 
-            ){
+        foreach ($arrTokens2 as $fieldName => $fieldDefinition) {
+            $objFormField = FormFieldModel::findOneBy(['pid = ?', 'name = ?'], [$arrTokens['formconfig_id'], $fieldName]);
+            if (!$objFormField
+            || $objFormField->is_technical_field
+            ) {
                 unset($arrTokens2[$fieldName]);
                 continue;
             }
 
             $arrTokens['useful_data_arr'][$fieldDefinition['label']] = $fieldDefinition['value'];
-            if(!empty($fieldDefinition['value'])){
+            if (!empty($fieldDefinition['value'])) {
                 $arrTokens['useful_data_filled_arr'][$fieldDefinition['label']] = $fieldDefinition['value'];
             }
         }
 
-        foreach($arrTokens['useful_data_arr'] as $label => $value){
-            $arrTokens['useful_data'].=sprintf("%s: %s\n",$label,$value);
+        foreach ($arrTokens['useful_data_arr'] as $label => $value) {
+            $arrTokens['useful_data'] .= sprintf('%s: %s<br />', $label, $value);
```

Comment:
We should split this in two vars, since we have a "text" field where \n should work and not <br>

Reply:
Now we have : 
- `useful_data` & `useful_data_filled` using `<br />`, available in `email_html` & `email_text` (only for compatibility)
- `useful_data_text` & `useful_data_filled_text` using `\n`, available in `email_text` only

Reply:
Once validated, I'll update the 1.0.x-rc2 branch